src/api/browse.ts

```diff
@@ -8,8 +8,9 @@ import { FacetFilterModel } from "../models/facets";
 import { fetchJson } from "../utils/utils";
 import { showMessage } from "../components/messages/usage";
 
-const MASS_URL = process.env.REACT_APP_MASS_URL;
-const METLDATA_URL = process.env.REACT_APP_METLDATA_URL;
+const CLIENT_URL = process.env.REACT_APP_CLIENT_URL
+const MASS_URL = (CLIENT_URL || "") + process.env.REACT_APP_MASS_URL;
+const METLDATA_URL = (CLIENT_URL || "") + process.env.REACT_APP_METLDATA_URL;
```

Comment:
We can assume that `CLIENT_URL` is always set.

We should support 3 different ways of configuring the `MASS_URL`: either as full URL with schema, or as absolute URL without schema but starting with a slash, or as relative URL. The concatenation to `CLIENT_URL` can be done using the [URL](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL) constructor: `mass_url = new URL (mass_url, client_url)`, then all of this is automatically supported.

Maybe we should also pass around urls not as strings, but as URL objects. Eventually they are used in the `fetch` call which also supports taking an `URL` object as a resource. This could be also refactored as part of this PR.